app/models/share/facebook.rb

```diff
@@ -2,24 +2,7 @@ class Share::Facebook < ActiveRecord::Base
   include Share::Variant
 
   validates :description, :title, presence: true
+  belongs_to :image
```

Comment:
My question was a little vague and should maybe be rephrased as to what happened to Paperclip here.